tests/unit/components/DocumentationTopic.spec.js

```diff
@@ -333,6 +338,13 @@ describe('DocumentationTopic', () => {
     expect(wrapper.find(DocumentationHero).find(Title).exists()).toBe(false);
   });
 
+  it('renders a `LinkableHeading`, in minimized mode, if `enableMinimized` prop is `true`', () => {
+    const heading = wrapper.find(LinkableHeading);
+    expect(heading.exists()).toBe(false);
+    wrapper.setProps({ enableMinimized: true });
+    expect(wrapper.find(LinkableHeading).exists()).toBe(true);
```

Comment:
You might also want to assert that a `Title` was not rendered since this is supposed to be taking its place in that scenario

Reply:
That's tested in the[ test for `Title`](https://github.com/apple/swift-docc-render/blob/67a60eb6a2f6908eba1374d88990e6070f1924bc/tests/unit/components/DocumentationTopic.spec.js#L336) from an earlier PR. I think keeping it there that makes more sense than moving it to the test for the `LinksableHeading`, what do you think?